README.md

```diff
@@ -147,13 +137,23 @@ The breakpoints are configured in `./src/scss/settings/_settings.breakpoints.scs
 ## File Generators
 To quickly add new parts to your app like components modules etc. the syntax to you can use is
 ```sh
-yo emakinacee-react:gen type name module
+react g <scaffold> <name> [module]
 ```
+
+Scaffold  | Usage
+---       | ---
+[Component](https://github.com/emakina-cee-oss/react-cli/wiki) | `react g component AwesomeComponent`
+[Module](https://github.com/emakina-cee-oss/react-cli/wiki)    | `react g module AwesomeModule`
+[Service](https://github.com/emakina-cee-oss/react-cli/wiki)   | `react g service AwesomeService`
+[Action](https://github.com/emakina-cee-oss/react-cli/wiki)    | `react g action awesomeAction SomeModule`
+[Factory](https://github.com/emakina-cee-oss/react-cli/wiki)   | `react g factory awesomeFactory SomeModule`
+[Compute](https://github.com/emakina-cee-oss/react-cli/wiki)   | `react g compute awesomeCompute SomeModule`
+
 See the following examples for more detail.
 
 ### Component
 ```sh
-yo emakinacee-react:gen component ComponentName
+react g component ComponentName
```

Comment:
I know that React components are written normally in PascalCase. In my opinion filenames with only lower case, have the advantage the you can not run into issues when you rename filenames, e.g. `MyComponentService` => `MycomponentService` vs. `my-component-service` => `mycomponent-service`.

Please write your thoughts, if we should follow the React convention or use the convention that e.g. Rails used.